### PR TITLE
Add more lsc colors

### PR DESCRIPTION
**Pull request by V1RU513N**

### CODE REVIEW

**Review by gir489returns — DISMISSED** [review]

File: src/core/data/lsc_types.hpp [hunk 1]
```diff
@@ -48,7 +48,7 @@ static const std::map<int, std::string> lsc_classic_colors = {
 	{COLOR_CLASSIC_BLACK, "Black"},
 	{COLOR_CLASSIC_CARBON_BLACK, "Carbon Black"},
 	{COLOR_CLASSIC_GRAPHITE, "Graphite"},
-	{COLOR_CLASSIC_ANHRACITE_GREY, "Anhracite Grey"},
+	{COLOR_CLASSIC_ANTHRACITE_BLACK, "Anhracite Black"},
```
Comment (gir489returns) [inline src/core/data/lsc_types.hpp]: According to the GTA V wiki, this is supposed to be Anthracite Black, yes?

**Review by gir489returns — CHANGES_REQUESTED** [review]
See: https://gta.fandom.com/wiki/Vehicle_Customization_in_GTA_V/Respray_Colors#Classic

File: src/core/data/lsc_types.hpp (hunk 1 above)
Comment (V1RU513N) [reply src/core/data/lsc_types.hpp]: Yes, the "color name" is “anthracite black” Anthracite grey would be the “technical name”, hence my confusion. I've chosen to stick to the “color name”.

https://docs.google.com/spreadsheets/d/e/2PACX-1vTa7yvIzGZRjK2cWS-jtkpD649KjV6vBR3cLjWQQ0JxCcLIotNGlxBMlqs_Z7c7lk8OQVUkGwgwdKqa/pubhtml

Comment (gir489returns) [reply src/core/data/lsc_types.hpp]: If you want to see how the game gets these values, you can use something like HUD::GET_FILENAME_FOR_AUDIO_CONVERSATION with the internal GXT strings to resolve it at runtime. This has the added benefit of using the game's translations for the colors.

If you need the GXT strings, carmod_shop should have them. I found a few using your spread sheet. func_1496 is Silver through Lime Green. func_1498 is Black through Frost White. func_1507 might be some kind of special color? Maybe pearlescent. It uses a couple of extra parameters, probably for the sheen percentage. It has Brushed Black Steel through Brushed Gold.

**Review by gir489returns — APPROVED** [review]

**Review by Yimura — APPROVED** [review]

**Pull request merged**